docs/web/contribute.md

```diff
@@ -2,7 +2,7 @@
 
 ## Set up a development environment
 
-1. Create a fork of the Ibis repository, and clone it.
+1. Create a fork of the [Ibis repository](https://github.com/ibis-project/ibis), and clone it.
```

Comment:
This change is from your other PR, you need to create branches. Now it's a bit tricky to fix. Make a copy of your local ibis repository first, in case something goes wrong, but something like this should fix it:
```
git fetch upstream
git merge upstream/master
git reset --soft upstream/master
git checkout -b move_pyspark_backend
git commit -m "Move pyspark backend to backends directory"
git push -u origin move_pyspark_backend
```
And then open a new PR. If you didn't set an `upstream` remote, you you'll have to do it with `git remote add upstream <main-ibis-repo-url>`
```